deno.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@db/mongo",
+  "version": "0.33.0",
+  "exports": {
+    ".": "./mod.ts",
+    "./client": "./src/client.ts",
+    "./collection": "./src/collection/collection.ts",
+    "./gridfs": "./src/gridfs/bucket.ts",
+    "./types": "./src/types.ts"
+  },
+  "publish": {
+    "exclude": [".github", "tests", ".gitattributes", ".gitignore", "deno.lock"]
+  },
+  "imports": {
+    "assert": "jsr:@std/assert@^0.220.1",
+    "b64": "jsr:@std/encoding@^0.220.1/base64",
+    "bdd": "jsr:@std/testing@^0.220.1/bdd",
+    "bytes_eq": "jsr:@std/bytes@^0.220.1/equals",
```

Comment:
Adding these import maps for a library is kinda a deal breaker for many stuff, often raw github is used as a cdn and this kinda breaks it, is there not a better way to do this? when jsr support import maps maybe make it only a opt in for jsr and still allowing raw github imports to function

Reply:
i'll go ahead and move the jsr specifiers back into a deps.ts file since that should allow the use of raw github without breaking anything on the jsr side

Reply:
moved in latest commit